oshi-core/src/main/java/oshi/software/os/ApplicationInfo.java

```diff
@@ -46,7 +47,7 @@ public ApplicationInfo(String name, String version, String vendor, long timestam
         this.version = version;
         this.vendor = vendor;
         this.timestamp = timestamp;
-        this.additionalInfo = additionalInfo != null ? new HashMap<>(additionalInfo) : new HashMap<>();
+        this.additionalInfo = additionalInfo != null ? new LinkedHashMap<>(additionalInfo) : new LinkedHashMap<>();
```

Comment:
```suggestion
        this.additionalInfo = additionalInfo != null ? new LinkedHashMap<>(additionalInfo) : Collections.emptyMap();
```
I'm not really sure we need the defensive copy here either, but why not keep at least one `Linked` in this class, I guess...

Reply:
Addressed in the latest commit